trytond/model/modelsql.py

```diff
@@ -1116,7 +1116,9 @@ def get_related_records(Model, field_name, sub_ids):
                         exception, {}, transaction=transaction)
                 raise
 
-        Translation.delete_ids(cls.__name__, 'model', ids)
+        translated = [f for f in cls._fields if getattr(f, 'translate', False)]
```

Comment:
Pas compris ;)

Reply:
Implem de base:
- delete record => delete potential translations (translatable field)
- delete record => delete translations if record class has translatable fields
=> No need to make db query when model is not translatable

Reply:
il faudrait cacher l'attribut sur la classe je pense

Reply:
L'idée est de fixer au plus simple et B2CK gère la suite